src/test/java/spoon/test/lambda/LambdaTest.java

```diff
@@ -403,6 +404,18 @@ public void testLambdaFilter() throws Exception {
 		assertHasStrings(methodNames, "m2", "m4", "m7", "m8");
 	}
 
+	@Test
+	public void testInterfaceWithObjectMethods() throws Exception {
+		CtInterface<?> checkPersons = factory.Interface().get(Foo.CheckPersons.class);
```

Comment:
please add contract to the first line of the test body. Something like
//contract: Lambda expression works on interfaces with methods inherited from java.lang.Object

Reply:
Done :)